src/applications/disability-benefits/526EZ/content/confirmation-page.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
maybe a silly thought but why the dash in the file name? Should it be consistent with the container name?

Reply:
I thought it might be weird to have capitals on a file that isn't exporting a component of the same name, but I'm open to whatever.